src/SchemaForm.vue

```diff
@@ -59,19 +56,43 @@ export default {
       type: Boolean,
       default: false
     },
+    nestedSchemaModel: {
+      type: String,
+      default: ''
+    },
     schemaRowClasses: {
       type: [String, Object, Array],
       default: null
     }
   },
   emits: ['submit', 'update:modelValue'],
-  setup (props, { emit }) {
+  setup (props, { emit, attrs }) {
+    const isChildOfWizard = inject('isSchemaWizard', false)
+
     const hasParentSchema = inject('parentSchemaExists', false)
     if (!hasParentSchema) {
       provide('parentSchemaExists', true)
     }
 
-    const { parsedSchema } = useParsedSchema(props)
+    const behaveLikeParentSchema = computed(() => (!isChildOfWizard && !hasParentSchema))
+
+    const { schema } = toRefs(props)
+    let injectedSchema = inject('injectedSchema', false)
```

Comment:
**suggestion:** Given that this is an OSS lib it might be good to use a more unique string or symbol for provide/inject to avoid accidental conflicts done by users that might not be aware of the internal workings of this library. Maybe something like a `fvl_` prefix?